co-authors-plus.php

```diff
@@ -1278,8 +1278,10 @@ function load_edit() {
 	 * @since 3.0
 	 */
 	function filter_views( $views ) {
+		$post_type = get_current_screen()->post_type;
```

Comment:
Since we're getting the post type already here, I think we can use the var `$mine_args['post_type']` from line 1294 and then, get rid of that entire conditional block in line 1293...so that we don't have to grab the post type twice.